pages/projects/search.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { useEffect, useState } from "react";
+import commonStyles from "../../styles/common.module.scss";
+import styles from "./Projects.module.scss";
+import Head from "next/head";
+import { Pagination, Typography } from "@equinor/eds-core-react";
+import { Layouts } from "../../layouts/LayoutWrapper";
+import { ProjectListSection } from "../../components/ProjectListSection";
+import { useQuery } from "react-query";
+import { getProjects } from "../../services/projectApi";
+import SideNavBar from "components/SideNavBar";
+import { checkIfCreatorOrEditor } from "utils/categorizeProjects";
+import { useAccount, useMsal } from "@azure/msal-react";
+import { debounce } from "utils/debounce";
+import SortMenu from "components/SortMenu";
+
+const itemsPerPage = 16; // increased from 19 to 100 since filtering (hiding project without name) is done on the client side
+// it looks bad when every other page just have a few cards and others have more... Therefore it would be better to just show a greater amount of cards at once
+
+export default function Projects(): JSX.Element {
+  const { accounts } = useMsal();
+  const account = useAccount(accounts[0]);
+  const [page, setPage] = useState(1);
+  const [q, setQ] = useState("");
```

Comment:
What is Q? Ref https://github.com/kettanaito/naming-cheatsheet#avoid-contractions